draft-ietf-core-coap-tcp-tls.md

```diff
@@ -588,8 +588,8 @@ protocol does not provide means for multiplexing. If it is not desirable for a
 large message to monopolize the connection, requests and responses can be
 transferred in a block-wise fashion as defined in {{I-D.ietf-core-block}}.
 
-Messages MUST NOT be Empty (Code 0.00), i.e., messages always carry
-either a request or a response.
+Empty messages (Code 0.00) are ignored by the recipient (see also

```

Comment:
Previous ADs might have complained of gratuitous RFC 2119 language, but yes, this is an interoperability requirement, so MUST is appropriate.